apps/cli-gui/src/main.ts

```diff
@@ -12,14 +12,14 @@ bootstrapApplication(AppComponent, {
   providers: [
     provideRouter(appRoutes, withEnabledBlockingInitialNavigation()),
     provideHttpClient(),
-    {
-      provide: APP_INITIALIZER,
-      useFactory: (http: HttpClient, core: CoreService) => () =>
-        http
-          .get<string[]>(`/api/workspace`)
-          .pipe(map(projectNames => core.update({ projectNames, currentProjectName: projectNames?.[0] }))),
-      deps: [HttpClient, CoreService],
-      multi: true,
-    },
+    // {
+    //   provide: APP_INITIALIZER,
+    //   useFactory: (http: HttpClient, core: CoreService) => () =>
+    //     http
+    //       .get<string[]>(`/api/workspace`)
+    //       .pipe(map(projectNames => core.update({ projectNames, currentProjectName: projectNames?.[0] }))),
+    //   deps: [HttpClient, CoreService],
+    //   multi: true,
+    // },
```

Comment:
I think you can uncomment this

Reply:
also convert to `inject` syntax instead of deps, also we need to extract the http endpoints to its own services

Reply:
@ronnetzer I'm not sure I understand what you mean regarding `inject`

Reply:
nvm its out of scope in any case... after migration to standalone we won't be able to use APP_INITIALIZER anyway so we'll refactor it when we do that